### ISSUE TITLE
Multiples questions

### ISSUE THREAD

**Issue by caywin25** [Ahwxorg/LibreY#187]
Hello and thank you for your work.

I will have some questions please.
Is it possible to add LibreY as default search engine in firefox?
I tried but it doesn't seem to work.

Also, is it possible to be able to do searches according to the language entered: 
example => :fr search or :en search?

Thanks for your answers

**Comment by Ahwxorg** [comment]
>  Is it possible to add LibreY as default search engine in firefox?

Yes, just right click the URL bar and click the button, shown in screenshot below;
![image](https://github.com/user-attachments/assets/262bf1d7-b240-4ed1-8f61-41ce56b7da19)

> Also, is it possible to be able to do searches according to the language entered:
> example => :fr search or :en search?

Currently not, you can use a lot of DDG bangs though, i.e. `!d hello` till take you to https://www.thefreedictionary.com/hello

**Comment by caywin25** [comment]
If I do your method to add LibreY as a search engine in Firefox I get this error: Firefox could not download the search plugin from: https://search.laifen.org/opensearch.xml?method=POST

**Comment by caywin25** [comment]
If I do your method to add LibreY as a search engine in Firefox I get this error: Firefox could not download the search plugin from: https://search_url.com/opensearch.xml?method=POST

**Comment by Ahwxorg** [comment]
> If I do your method to add LibreY as a search engine in Firefox I get this error: Firefox could not download the search plugin from: https://search_url.com/opensearch.xml?method=POST

Could this be something like #128?

> Thanks for the bangs.
>  Are there any "internal to libreY" bangs that would allow searching by doing for example: !images and that would go directly to the image section of LibreY?

They do not exist I believe, but should be easy to add. Take a look at https://github.com/Ahwxorg/LibreY/blob/main/static/misc/ddg_bang.json

**Comment by caywin25** [comment]
> Could his be something like https://github.com/Ahwxorg/LibreY/issues/128?

Yes exactely

> They do not exist I believe, but should be easy to add. Take a look at https://github.com/Ahwxorg/LibreY/blob/main/static/misc/ddg_bang.json

Thank you

**Comment by caywin25** [comment]
> Could his be something like https://github.com/Ahwxorg/LibreY/issues/128?

I tried to follow the steps as said in #128 but I didn't succeed
Could you help me please?
Thanks

**Comment by Ahwxorg** [comment]
I really dont know what issue you are having, since I can't reproduce it.

**Comment by PardesiCat** [comment]
Hi  am not sure but this maybe reason 

in your main instance https://search.ahwx.org/opensearch.xml


insead of instance url its showing localhost ip


but in your other instance https://search2.ahwx.org/opensearch.xml  https://search3.ahwx.org/opensearch.xml

url set to  "https://search.ahwx.org/opensearch.xml?method=GET"

![image](https://github.com/user-attachments/assets/b08a78bb-cb96-4737-955b-5a477dbbe531)

**Comment by Ahwxorg** [comment]
That's true, and weird, I'll look into that.

**Comment by Ahwxorg** [comment]
Fixed, for everyone on Docker, you can use the environment variable OPEN_SEARCH_HOST (or map opensearch.xml to `/var/www/html/opensearch.xml`).

**Comment by caywin25** [comment]
Hello

My initial issue is solved.
I think we can close this issue :)
Thanks
